products/django.md

```diff
@@ -58,6 +58,18 @@ releases:
 
 > [Django](https://www.djangoproject.com/) is a high-level Python Web framework that encourages rapid development and clean, pragmatic design.
 
+**Python Support**
+
+Django 4.0 supports Python 3.8, 3.9, and 3.10
+
+Django 3.2 supports Python 3.6, 3.7, 3.8, 3.9, and 3.10 (as of 3.2.9)
+
+Django 2.2 supports Python 3.5, 3.6, 3.7, 3.8 (as of 2.2.8), and 3.9 (as of 2.2.17)
+
+Django 2.0 supports Python 3.4, 3.5, 3.6, and 3.7
+
+Django 1.11.x series is the last to support Python 2.7
+
```

Comment:
```suggestion
Django Release|Supported Python Versions
--------------|-------------------------
4.0           |3.8-3.10
3.2           |3.6-3.10
2.2           |3.5-3.9
2.0           |3.4-3.7
1.11.x        |2.7,3.4-3.7

Not all releases in a series will support all Python versions, the matrix above is for the latest patch release.
```

Is there a link that we could add here? Or mention where they can find this information readily upstream? Perhaps link each link to the corresponding release note again?

Reply:
This table looks much better but i still insist about not giving it with intervals like a.b - y.z but writing all possible values  a.b , c.d , y.x

Reply:
I have taken all these values one by one from each version's changelog like : 
https://docs.djangoproject.com/en/4.0/releases/2.0/#python-compatibility
https://docs.djangoproject.com/en/4.0/releases/2.2/#python-compatibility
https://docs.djangoproject.com/en/4.0/releases/3.2/#python-compatibility
https://docs.djangoproject.com/en/4.0/releases/4.0/#python-compatibility